ConsoleReporter.coffee

```diff
@@ -113,5 +113,5 @@ class ConsoleReporter extends  practical.mocha.BaseReporter
     finally
       log.return()
 
-Meteor.startup ->
```

Comment:
We've slightly change the way test drivers work in 1.3 -- the idea is that you'd `meteor add` one or more of them into your app and then pick the one you want at test time with `--driver-package`. This works now thanks to the `testOnly` package flag.

The advantage of this way is that you can choose the version of your driver package in a reasonable way.